action.yml

```diff
@@ -32,9 +32,12 @@ inputs:
 runs:
   using: "composite"
   steps:
-    - name: Install PyYAML
+    - name: Setup python venv and dependencies
       shell: bash
-      run: pip install --user PyYAML
+      run: |
+        python3 -m venv .venv
+        source .venv/bin/activate
+        python3 -m pip install PyYAML
```

Comment:
You need to share the same shell session between the setup and the Clifus run to make this work

Reply:
otherwise VIRTUALENV_PATH will not be set in next shell session.

Reply:
oh :(

Reply:
If we create the virtualenv in the origin repository does it work ?

Reply:
Nevermind you meant between steps in here i guess, modified the PR accordingly